dps_shared.def

```diff
@@ -29,7 +29,6 @@ DPS_PublicationGetSequenceNum
 DPS_PublicationGetTopic
 DPS_PublicationGetUUID
 DPS_Publish
-DPS_Rand
```

Comment:
Is DPS_Rand no longer part of the public API. I did not see that it was removed from the code.